lib/vscode/src/vs/platform/environment/common/environmentService.ts

```diff
@@ -173,6 +190,19 @@ export abstract class AbstractNativeEnvironmentService implements INativeEnviron
 		return undefined;
 	}
 
+	/**
+	 * NOTE@coder: add extraExtensionPaths and extraBuiltinExtensionPaths
+	 * Code location changed after 1.54 (was earlier directly in NativeEnvironmentService).
```

Comment:
Just a nit, but I think we can put the portion here about what changed in the commit message (or in this case leave it out since the commit already shows it moved). 

My reasoning is that someone coming in here and reading this wouldn't have the context required to understand it without the commit anyway.

Plus if we follow this pattern we would end up maintaining a changelog in the comments. :laughing: